doc/man/dash-qt.1

```diff
@@ -1029,7 +1088,7 @@ Set the font weight for bold texts. Possible range 0 to 8 (default: 4)
 .HP
 \fB\-font\-weight\-normal\fR
 .IP
-Set the font weight for normal texts. Possible range 0 to 8 (default: 2)
+Set the font weight for normal texts. Possible range 0 to 8 (default: 1)
```

Comment:
UPD:
it seems as it depends from 2 things:
 - your QT version (if that ever changed)
 - is it MAC or not MAC build

```
#ifdef Q_OS_MAC
static const QFont::Weight defaultFontWeightNormal = QFont::ExtraLight;
static const QFont::Weight defaultFontWeightBold = QFont::Medium;
static const int defaultFontScale = 0;
#else       
static const QFont::Weight defaultFontWeightNormal = QFont::Light;
static const QFont::Weight defaultFontWeightBold = QFont::Medium;
static const int defaultFontScale = 0;
#endif  
```

so, you have 'default:1` because you collected this data on mac; otherwise it should be 2.
Let's revert this changes so far as probably amount of `windows + linux > mac` (my assumption).

Reply:
```suggestion
Set the font weight for normal texts. Possible range 0 to 8 (default: 2)
```